### PR TITLE
Feature/menu

### PR DESCRIPTION
**Pull request by akaFTS**
#### What is the purpose of this pull request?
This PR implements basic navigation for the app, along with basic page structure and i18n configuration. Responsive menu layout is also working as desired. Everything is still fully static, though, and "orders" menu won't work for a while.

#### Screenshots or example usage
image

image

image

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Menu/UserInfo.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                <img className="br-100 w3" src={userPlaceholder} />
+                <span className="absolute bottom-0 right-0 blue">
+                    <Plus size={20} color="currentColor"/>
+                </span>
+            </div>
+            <div>
+                <div className="f5 fw3 helvetica mid-gray mb2">
+                    { intl.formatMessage({id: 'userInfo.greeting'}) },
+                </div>
+                <div className="f4 fw3 helvetica">Gustavo!</div>
```
Comment (analuizamtg) [inline react/components/Menu/UserInfo.js]: hard-coded

File: react/components/Menu/Menu.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import classnames from 'classnames'
+import PropTypes from 'prop-types';
+import UserInfo from './UserInfo'
+import MenuLinksList from './MenuLinksList'
+
+const Menu = ({ className }) => {
```
Comment (klzns) [inline react/components/Menu/Menu.js]: Acho que nao precisa dessa prop, o componente em seu unico uso recebe o valor "mr8". Pode colocar direto no componente.

File: react/components/Menu/MenuLinksList.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { injectIntl, intlShape } from 'react-intl'
+import MenuLink from './MenuLink';
+
+const links = [
+    {code: 'pages.orders', path: '#'},
+    {code: 'pages.addresses', path: '/addresses'},
+    {code: 'pages.personalData', path: '/personal_data'},
+    {code: 'pages.paymentData', path: '/payment_data'}
```
Comment (klzns) [inline react/components/Menu/MenuLinksList.js]: O que acha de usar `id` ao inves de `code`? O nome fica mais alinhado o parametro esperado pela lib de traducao.

File: react/components/Menu/UserInfo.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl' 
+import classnames from 'classnames'
+import Plus from '@vtex/styleguide/lib/icon/Plus'
+import userPlaceholder from '../../images/user-placeholder.svg'
+
+const UserInfo = ({ className, intl }) => {
```
Comment (klzns) [inline react/components/Menu/UserInfo.js]: Evitar criar componentes que recebam `className`. Isso os deixam imprevisiveis quanto ao que renderizam.

File: react/components/Menu/UserInfo.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl' 
+import classnames from 'classnames'
+import Plus from '@vtex/styleguide/lib/icon/Plus'
+import userPlaceholder from '../../images/user-placeholder.svg'
```
Comment (klzns) [inline react/components/Menu/UserInfo.js]: Preferir fazer um componente que renderize o svg do que importar o arquivo '.svg'.

File: react/theme.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+/* Complementing Tachyons */
+
+.helvetica {
+    font-family: 'helvetica neue', helvetica, sans-serif;
+}
+
+/* As of 07/2018, @vtex/styleguide icons can't receive tachyon classes */
+.user-picture svg{
```
Comment (klzns) [inline react/theme.css]: Se vc criar um componente React pro svg, vc pode colocar as cores sem usar CSS.

**Pull request merged**